osert.rb

```diff
@@ -284,7 +284,7 @@ def sed(file, text_i, text_o)
 
     # Generating report
     puts '[+] Generating report...'
-    pdf = "#{output}/#{exam}-#{osid}-Exam-Report.pdf"
+    pdf = "#{output}/#{exam}-OS-#{osid}-Exam-Report.pdf"
```

Comment:
The `OS-` prefix is already included in the variable `osid` see https://github.com/noraj/OSCP-Exam-Report-Template-Markdown/blob/2e245f8ff07664d08bfad6a493323776bfa05431/osert.rb#L208